### ISSUE TITLE
AgoraRTMModule.java:3 - Error - package android.support.annotation does not exist

### ISSUE THREAD

**Issue by fritzfr** [AgoraIO/agora-react-native-rtm#17]
I am compiling my app via Bitrise and I am getting this compilation error:

> Task :agora-react-native-rtm:compileReleaseJavaWithJavac FAILED
/bitrise/src/mobile-app/node_modules/agora-react-native-rtm/android/src/main/java/io/agora/agora_rtm/AgoraRTMModule.java:3: error: package android.support.annotation does not exist
import android.support.annotation.Nullable;
                                 ^
/bitrise/src/mobile-app/node_modules/agora-react-native-rtm/android/src/main/java/io/agora/agora_rtm/AgoraRTMModule.java:73: error: cannot find symbol
    private void sendEvent(String eventName, @Nullable WritableMap params) {
                                              ^
  symbol:   class Nullable
  location: class AgoraRTMModule
2 errors
FAILURE: Build failed with an exception.

which comes from here: 
https://github.com/AgoraIO/agora-react-native-rtm/blob/b99c88bc8958d9e9e56db82575cc5f7477c6ca65/android/src/main/java/io/agora/agora_rtm/AgoraRTMModule.java#L3

Any ideas what the problem is? I have androidX and jetifier enabled:

```
android.useAndroidX=true
# Automatically convert third-party libraries to use AndroidX
android.enableJetifier=true
```

**Comment by fritzfr** [comment]
I "fixed" this by manually executing `npx jetify` before making the build. I think you should migrate this line though.

**Comment by LichKing-2234** [comment]
I will remove this annotation next version.

**Comment by LichKing-2234** [comment]
fix in 1.2.2-alpha.2
